cmd/influxd/config.go

```diff
@@ -60,18 +87,39 @@ var DefaultSnapshotURL = url.URL{
 	Host:   net.JoinHostPort(DefaultSnapshotBindAddress, strconv.Itoa(DefaultSnapshotPort)),
 }
 
+// Broker represents the configuration for a broker node
+type Broker struct {
+	Dir     string   `toml:"dir"`
+	Enabled bool     `toml:"enabled"`
+	Timeout Duration `toml:"election-timeout"`
+}
+
+// Snapshot represents the configuration for a snapshot service
+type Snapshot struct {
```

Comment:
I wonder if it would be worth moving this block into `data`, somewhat like `Retention` since snapshotting can only be performed on data nodes.